storage/etcd/storage.go

```diff
@@ -89,7 +90,10 @@ func (s *Storage) List(key string, objsPtr interface{}) error {
 	for _, kv := range resp.Kvs {
 		// Decode and append the value to v, which must be a slice.
 		// See https://github.com/kubernetes/apiserver/blob/10d97565493b4eea44b1ef6c1b3fd47d2876a866/pkg/storage/etcd3/store.go#L786
-		obj := reflect.New(v.Type().Elem()).Interface().(meta.Object)
+		obj, ok := reflect.New(v.Type().Elem()).Interface().(meta.Object)
```

Comment:
Need to cast to *meta.Object

Reply:
Not sure to understand why; `reflect.New(v.Type().Elem())` returns a `*simple.Simple`, that we cast to a `meta.Object` interface. Using `(*meta.Object)` would cause the cast to fail.